pom.xml

```diff
@@ -2088,7 +2088,7 @@
       <dependency>
         <groupId>${hive.group}</groupId>
         <artifactId>hive-service-rpc</artifactId>
-        <version>${hive.version}</version>
+        <version>3.1.2</version>
```

Comment:
Shall we have a JIRA to recover this when we upgrade Hive version to 3.1.2?

Reply:
ok

Reply:
@wangyum . Did you create a JIRA? I cannot find it yet. If you created, please link to SPARK-33525 .

Reply:
Yes. it's https://issues.apache.org/jira/browse/SPARK-33550